packages/map-template/src/components/Map/Map.jsx

```diff
@@ -12,6 +12,12 @@ const MAP_TYPES = {
 
 const localStorageKeyForVenue = 'MI-MAP-TEMPLATE-LAST-VENUE';
 
+/**
+ * Private variable used for storing the tile style.
+ * Implemented due to the impossibility to use the React useState hook.
+ */
+let _tileStyle;
```

Comment:
Do we need this at all? Can't we just pass the tileStyle to onTileStyleChanged from the useEffect hook?

Reply:
No, I don't think we need it anymore. I have removed it and used the `tileStyle` instead.